src/Options.php

```diff
@@ -641,6 +641,30 @@ public function setHttpProxy(?string $httpProxy): void
         $this->options = $this->resolver->resolve($options);
     }
 
+    /**
+     * Return the capture_silenced_errors option.
+     *
+     * @return bool if true, errors silenced through the @ operator will be reported,
+     *              ignored otherwise
+     */
+    public function shouldCaptureSilencedErrors(): bool
+    {
+        return $this->options['capture_silenced_errors'];
+    }
+
+    /**
+     * Sets the capture_silenced_errors option.
```

Comment:
This sentence should describe what the option does rather than just say that it sets the option named X. I would change it to something like `Sets whether the silenced errors should be captured or not`